siliconcompiler/core.py

```diff
@@ -2404,7 +2409,12 @@ def archive(self, step=None, index=None, all_files=False):
         Args:
             step(str): Step to archive.
             index (str): Index to archive
+            nodelist (list): List of step/index pairs to include in the archive
             all_files (bool): If True, all files are archived.
+            archive_name (str): Path to the archive
+            include_manifest (bool): If True, the final manifest will be included
+            additional_files (dict of str): Dictionary with additional files to include
+                in the archive with their associated filename in the archive
```

Comment:
Maybe reword a little to make it clear whether keys or values are "real" filenames vs filenames in the tarball - took me a bit to parse the sentence